apps/frontend/src/app/5_pages/ZeroPage/utils/trove-manager.ts

```diff
@@ -37,7 +73,7 @@ export const openTrove = async (
   return {
     value: value.hex,
     fn: token === SupportedTokens.dllr ? 'openNueTrove' : 'openTrove',
-    args: [maxBorrowingRate.hex, borrowZUSD.hex, ...hints],
+    args: [fees?.maxBorrowingFeeRate, borrowZUSD.hex, ...hints],
```

Comment:
```suggestion
    args: [fees?.maxBorrowingFeeRate.toHexString(), borrowZUSD.hex, ...hints],
```
I think it's decimal now, so make sure you are passing hex to contract